app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons_list = [Person(person["name"], person["age"]) for person in people]
+    for name, person in Person.people.items():
+        for personal_info in people:
+            if personal_info.get("wife") and name == personal_info["name"]:
+                person.wife = Person.people[personal_info["wife"]]
+            if personal_info.get("husband") and name == personal_info["name"]:
+                person.husband = Person.people[personal_info["husband"]]
```

Comment:
you can simplify this solution. No need in nested loops

Reply:
thanks for remark, now its easier to read, i believe...
can you also advice please, is it a "bad practice" to make empty string before both "if" ?
anyway, string have some space because of loop, but its gonna be more readable on my mind

Reply:
No it is not a bad practice. It depends on specification of the project you working on. In this case - it is up to you